### PR TITLE
Simplify Git in Visual Studio page

### LINKED ISSUE #1451: Section on using git within Visual Studio is 6 years out of date
**Which version of the book is affected?**

The version that's present on `master`.

**Describe the bug:**

https://github.com/progit/progit2/blob/master/book/A-git-in-other-environments/sections/visualstudio.asc
is out of date, and contains outdated pictures.

**Steps to reproduce:**

1. Go to: https://github.com/progit/progit2/blob/master/book/A-git-in-other-environments/sections/visualstudio.asc
2. Notice last commit was 6 years ago.
3. Notice picture are outdated.

**Expected behavior:**

1. Section up to date with current version of Visual Studio 2019 **or** no Progit2 custom section at all, just a link to current docs for Visual Studio 2019.

**Screenshots:**

**Additional context:**

**Desktop:**

- Operating system:
- Browser/application:
- Browser/application version:

**Smartphone:**

- Device:
- OS:
- Browser/application:
- Browser/application version:

**E-book reader:**

- Device:
- Software Update:

---

# Discussion:

The outdated Visual Studio section was also noted in #1205.

EDIT: The dead link that I originally reported is fixed with the merge of #1450.
I also updated this issue report to take that into account, as not to confuse readers after the merge. 😄

### PR DESCRIPTION
**Pull request by prnadago**
- [x] I provide my work under the [project license](https://github.com/progit/progit2/blob/master/LICENSE.asc).
- [x] I grant such license of my work as is required for the purposes of future print editions to [Ben Straub](https://github.com/ben) and [Scott Chacon](https://github.com/schacon).

## Changes
The current page has outdated content about Team Explorer. The default Git tooling in Visual Studio has been updated. It's probably best to simplify this page to briefly highlight the Git functionality and point readers to the official docs where they can read a lot more about the details. 

I've also removed the screenshots since the team is going to continue enhancing the Git tooling in VS for at least the next year. The screenshots are going to get outdated pretty fast. The link to the Visual Studio documentation will stay up to date with the latest screenshots and functionality. 

## Context
Fixes #1451

### CODE REVIEW

File: book/A-git-in-other-environments/sections/visualstudio.asc
Comment (HonkingGoose) [inline book/A-git-in-other-environments/sections/visualstudio.asc:18]:
```suggestion
Read the https://docs.microsoft.com/visualstudio/version-control[official documentation^] to learn more.
```

How about this sentence?

I'm using the "caret shorthand" to open the external link in a new browser window. See https://asciidoctor.org/docs/asciidoc-writers-guide/#target-window-and-role-attributes-for-links.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]
Looks good! I agree that images go out of date fast, especially those of interfaces, so best not to bother with images.

I have one thing that we could improve, see my suggestion for that change.

File: book/A-git-in-other-environments/sections/visualstudio.asc
Comment (prnadago) [reply book/A-git-in-other-environments/sections/visualstudio.asc:18]: Yep I like it!

**Review by HonkingGoose — APPROVED** [review]

**Comment by prnadago** [comment]
@HonkingGoose thanks for approving. Anything else I need to do to get this merged?

**Comment by HonkingGoose** [comment]
This needs a review/merge from somebody with write level rights. I only have read right so I can't merge this PR. 😉 

I'll ping @ben (maintainer of this repository) for a review.

**Comment by ben** [comment]
Fabulous, thanks!

**Pull request merged**
